src/systemctl/systemctl.c

```diff
@@ -4245,10 +4245,15 @@ static void print_status_info(
                         printf(" Main PID: "PID_FMT, i->main_pid);
 
                         if (i->running) {
-                                _cleanup_free_ char *comm = NULL;
-                                (void) get_process_comm(i->main_pid, &comm);
-                                if (comm)
-                                        printf(" (%s)", comm);
+
+                                if (arg_transport == BUS_TRANSPORT_LOCAL) {
```

Comment:
`if (i->running && arg_transport == BUS_TRANSPORT_LOCAL) {`

Reply:
well, the "else" branch doesn't permit that...

Reply:
Oh, you mean, if running, then exit_code is positive? OK. please ignore my previous comment.